src/Components/SourcesSelect/index.js

```diff
@@ -26,7 +26,7 @@ const SourcesSelect = ({ setValidation, imageSourceID }) => {
       const id = chosenSource;
 
       if (IB_SOURCE_PROVIDERS.includes(provider) && imageSourceID) {
-        setSelected(selectObject(imageSourceID, data.find((source) => source.id === imageSourceID).name));
+        setSelected(selectObject(imageSourceID, data.find((source) => source.id == imageSourceID).name));
```

Comment:
FYI I'm removing this code in #246
If you'd not mind I guess it would be better to drop this and related changes, to reduce conflicts.